client-v2/src/components/Clipboard.tsx

```diff
@@ -153,112 +133,99 @@ class Clipboard extends React.Component<ClipboardProps> {
 
   public render() {
     return (
-      <ClipboardWrapper
-        tabIndex={0}
-        onFocus={this.handleFocus}
-        onBlur={this.handleBlur}
-        innerRef={this.clipboardWrapper as Ref}
-      >
-        <StyledDragIntentContainer
-          active={!this.props.isClipboardOpen}
-          delay={300}
-          onDragIntentStart={() => this.setState({ preActive: true })}
-          onDragIntentEnd={() => this.setState({ preActive: false })}
-          onIntentConfirm={() => this.props.toggleClipboard(true)}
-        >
-          <ClipboardHeader>
-            {this.props.isClipboardOpen && (
-              <ClipboardTitle>Clipboard</ClipboardTitle>
-            )}
-            <ButtonCircularCaret
-              tabIndex={-1}
-              openDir="right"
-              active={this.props.isClipboardOpen}
-              preActive={this.state.preActive}
-              onClick={() =>
-                this.props.toggleClipboard(!this.props.isClipboardOpen)
-              }
-            />
-          </ClipboardHeader>
-          <ClipboardBody>
-            {this.props.isClipboardOpen && (
-              <Root
-                id="clipboard"
-                data-testid="clipboard"
-                style={{ display: 'flex', flex: 1 }}
-              >
-                <ClipboardLevel
-                  onMove={this.handleMove}
-                  onDrop={this.handleInsert}
+      <React.Fragment>
+        {this.props.isClipboardOpen && (
```

Comment:
Super minor and abstract point that probably isn't worth changing in this PR -- is this where we'd want to add this condition? It makes our clipboard component unusable elsewhere. As a rule of thumb I've found that if you're writing a conditional at the root of a component's render fn, it probably belongs somewhere further up the tree. I'll stop writing now ahhh